build.gradle

```diff
@@ -1,21 +1,45 @@
-plugins {
-    id 'com.jfrog.bintray' version '1.6'
-    id 'nebula.optional-base' version '3.0.3'
-    id 'pl.allegro.tech.build.axion-release' version '1.3.4'
+buildscript {
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
+    dependencies {
+        classpath "com.ofg:micro-common-release:0.1.18"
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.6"
+        classpath "com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3"
+    }
+    dependencies {
+        ant.unjar src: configurations.classpath.find { it.name.startsWith("micro-common-release") }, dest: 'build/release'
+    }
 }
 
-apply plugin: 'groovy'
-apply plugin: 'nebula.provided-base'
-apply plugin: 'maven-publish'
+apply from: "${rootProject.buildDir}/release/gradle/version.gradle"
 
 scmVersion {
-    ignoreUncommittedChanges = false
-
     tag {
-        prefix = ''
+        prefix = 'release'
+        versionSeparator = '/'
     }
 }
 
+group = "io.fourfinanceit.pipeline"
+version = scmVersion.version
+
+apply plugin: 'groovy'
+apply from: "${rootProject.buildDir}/release/gradle/publish.gradle"
+apply from: "${rootProject.buildDir}/release/gradle/release.gradle"
+
+bintray {
+    pkg {
+        repo = 'jenkins-pipeline'
+        vcsUrl = 'https://github.com/4finance/jenkins-pipeline-dsl.git'
+        labels = ['jenkins', 'pipeline', 'ci', 'cd', 'continuous-integration', 'continuous-delivery']
+    }
+    publish = true
```

Comment:
It should be removed (is set to `false` by default). `finalizeRelease` task does it after successful push to GH from Travis.